ray-operator/apis/ray/v1alpha1/raycluster_types.go

```diff
@@ -56,6 +58,8 @@ type WorkerGroupSpec struct {
 	Template v1.PodTemplateSpec `json:"template"`
 	// ScaleStrategy defines which pods to remove
 	ScaleStrategy ScaleStrategy `json:"scaleStrategy,omitempty"`
+	// RayResources is a string-int map specifying Ray custom resources and overrides for CPU,GPU,memory.
+	RayResources RayResources `json:"rayResources,omitempty"`
```

Comment:
Is this OK - to use the type name for the field name? I do see this pattern at a few other locations in this file though.